src/haliax/__init__.py

```diff
@@ -36,6 +36,7 @@
 from .hof import fold, map, scan, vmap
 from .ops import clip, isclose, pad_left, trace, tril, triu, where
 from .partitioning import auto_sharded, axis_mapping, fsdp, named_jit, shard_with_axis_mapping
+from .specialized_fns import top_k
```

Comment:
you should add top_k to __all__. sorry i missed that.

Reply:
Ah, didn't realize I missed that. Just added it.